test/test_helper.rb

```diff
@@ -14,8 +15,12 @@
 }
 GlobalUid::Base.extend(GlobalUid::ServerVariables)
 
-yaml = YAML.load(IO.read(File.dirname(__FILE__) + "/config/database.yml"))
-ActiveRecord::Base.configurations = yaml
+Phenix.configure do |config|
+  config.database_config_path = File.join(File.dirname(__FILE__), "config/database.yml")
+end
+
+Phenix.rise!(with_schema: false)
 ActiveRecord::Base.establish_connection(:test)
```

Comment:
should connecting be handled inside of phenix ? / can we avoid the duplicate .rise! ?